### DISCUSSION TITLE
no zoom on photos for shared album ?

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#3808] [Q&A]
Hello, 
I cannot find how to zoom on photos when sharing an album with a public link.
When using the shared link, if the user does a left click on a photo, nothing appens, while i was expecting to view the photo in full size.
Is it the expected behavior ?

**Comment by jrasm91** [comment]
It is a bug and should be fixed with
https://github.com/immich-app/immich/pull/3791
